transcom/mymove
# [ MB-16759 ] Removing the departure date validation check in the number days SIT for additional days SIT service items

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16759)

## Summary

This PR removes the validation check for departure date on payment requests relating to Origin and Destination SIT service items in the number of days SIT lookup. This validation check is only related to first day and additional days service items. 

### What was happening
The error was occurring when a payment request for a first day origin or destination SIT service item that has a departure date was made before a payment request for additional days SIT. We would hit this validation check and the payment request for the additional days SIT could not get created. 

### How to replicate the error
1. As the prime go to an approved move and create either origin or destination SIT service items, make sure to include the departure date.
2. As a TOO go to that move and approve those service items.
3. As the prime create a payment request for the first day SIT service item. This should get approved
4. As the prime create a payment request for the additional days SIT service item. This should throw the error


## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. As the prime go to an approved move and create either origin or destination SIT service items, make sure to include the departure date.
2. As a TOO go to that move and approve those service items.
3. As the prime create a payment request for the first day SIT service item. This should get approved
4. As the prime create a payment request for the additional days SIT service item. You should not see the error.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
